test/ThriftTest.thrift

```diff
@@ -86,7 +86,7 @@ struct Xtruct
 
 struct Xtruct2
 {
-  1: byte   byte_thing,
+  1: i8     byte_thing,  // used to be byte, hence the name
```

Comment:
Agree, but one of my goals was to break it down into small changes to prevent us from ending up with a monster patch. This could be one of the next steps.